terracotta/src/test/java/org/ehcache/ClusteredCacheManagerTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache;
+
+import org.ehcache.config.BaseCacheConfiguration;
+import org.ehcache.config.BaseClusteredCacheSharedConfiguration;
+import org.ehcache.config.CacheRuntimeConfiguration;
+import org.ehcache.config.ClusteredCacheSharedConfiguration;
+import org.ehcache.config.ClusteredCacheSharedRuntimeConfiguration;
+import org.ehcache.config.EvictionPrioritizer;
+import org.ehcache.config.EvictionVeto;
+import org.ehcache.config.ResourcePoolsBuilder;
+import org.ehcache.config.TerracottaBaseCacheConfiguration;
+import org.ehcache.config.TerracottaCacheRuntimeConfiguration;
+import org.ehcache.config.TerracottaConfiguration;
+import org.ehcache.config.units.EntryUnit;
+import org.ehcache.config.units.MemoryUnit;
+import org.ehcache.expiry.Expiry;
+import org.junit.Test;
+
+import java.net.URI;
+
+/**
+ * @author Alex Snaps
+ */
+public class ClusteredCacheManagerTest {
+
+  @Test
+  public void testNothingButAPI() {
+
+    final URI terracottaURI = sourceThat();
+    final TerracottaConfiguration tcConfig = new TerracottaConfiguration(terracottaURI);
+    tcConfig.setDefaultCacheSize(42, MemoryUnit.GB, true); // not saying we should have this default here or at all
```

Comment:
I think I'd rather have a default `Pool` a clustered `CacheManager` instance uses for it's clustered `Cache` instances by default. But I guess we really have to get going with #40 et al for that.